pegasus/sites.v3/code.org/public/css/volunteer-local.css

```diff
@@ -85,3 +85,7 @@ html, body, #map-canvas {
 .no-margin {
   margin: 0;
 }
+
+.profile-detail {
+  white-space: pre-line;
```

Comment:
Very handy.  afaict from a quick search, this works on all the browsers we care about?